app-config

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+set +u
+NAME=$NAME
```

Comment:
This line itself does nothing (other than throw an exception if the variable is not set) With the introduction of `utils`, can we push the NAME creation there?

Reply:
Now that I managed to get things to work with the recent changes I will do a review of the various sections and readme.